### DISCUSSION TITLE
Software and database version

### DISCUSSION THREAD

**Discussion by francklap** [osm-search/Nominatim#2366] [Q&A]
Hello,

I recently installed a new server with Nominatim. I installed it with version 3.7.1, and upgraded to version 3.7.2. But in the status page, I always had this information for both version:
Software version
    3.7.0-0
Database version
    3.7.0-0

Also this is the output of the migrate command:
nominatim admin --migrate
2021-06-17 05:10:13: Using project directory: /data/nominatim
2021-06-17 05:10:13: Checking for necessary database migrations
2021-06-17 05:10:13: Database already at latest version (3.7.0-0)

Did I miss something or is it normal that it said v3.7.0 instead of 3.7.2? How to be sure I correctly upgraded to version 3.7.2?

Thanks.

**Comment by lonvia** [comment]
That's a bit of an open issue. I did not increase the minor version for the 3.71 and 3.7.2 releases because I thought it would have unintended side-effects on the migrations if database and software versions diverge between the 3.7 branch and master. That was probably not so correct. In any case, we'll get this sorted out in time for the next major release.
